app/controllers/api/motions_controller.rb

```diff
@@ -24,6 +24,12 @@ def update_outcome
     respond_with_resource
   end
 
+  def closed
+    load_and_authorize :group
+    @motions = @group.closed_motions.order(closed_at: :desc)
```

Comment:
Thanks for explaining this James. For some reason I've had a hard time understanding this in the past.

Lemme see if I've got it right:
visible_records defines the collection
instantiate_collection applies any conditions you want

?